SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -121,6 +121,24 @@ class CVariable {
   su2matrix<int> AD_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
   su2matrix<int> AD_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
 
+  su2matrix<int> AD_Time_n_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Time_n_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
+
+  su2matrix<int> AD_Time_n1_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Time_n1_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
+
+  su2matrix<int> AD_Vel_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Vel_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
+
+  su2matrix<int> AD_Vel_Time_n_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Vel_Time_n_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
+
+  su2matrix<int> AD_Accel_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Accel_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
+
+  su2matrix<int> AD_Accel_Time_n_InputIndex;    /*!< \brief Indices of Solution variables in the adjoint vector. */
+  su2matrix<int> AD_Accel_Time_n_OutputIndex;   /*!< \brief Indices of Solution variables in the adjoint vector after having been updated. */
```

Comment:
If these indices are for the FEA variables only they should probably be moved there to avoid allocating them unnecessarily.

Reply:
Yes, they have been moved